internal/cli/backgroundassets/background_assets_submit.go

```
@@ -0,0 +1,471 @@
+
+			for i, item := range items {
+				if _, skip := alreadyAttached[item.BackgroundAssetVersionID]; skip {
+					result.SkippedAlreadyAttached = append(result.SkippedAlreadyAttached, item)
+					continue
+				}
+				if _, err := client.CreateReviewSubmissionItem(requestCtx, currentSubmissionID, asc.ReviewSubmissionItemTypeBackgroundAssetVersion, item.BackgroundAssetVersionID); err != nil {
+					if createdHere {
+						if _, cancelErr := client.CancelReviewSubmission(requestCtx, currentSubmissionID); cancelErr == nil {
+							return fmt.Errorf("background-assets submit: attach version %q (index %d, %d already attached) to submission %q failed; rolled back the submission: %w", item.BackgroundAssetVersionID, i, result.AttachedItems, currentSubmissionID, err)
+						} else {
+							return fmt.Errorf("background-assets submit: attach version %q (index %d, %d already attached) to submission %q failed; rollback also failed (submission %q is leaked with %d partial item(s)): attach err=%v; rollback err=%v", item.BackgroundAssetVersionID, i, result.AttachedItems, currentSubmissionID, currentSubmissionID, result.AttachedItems, err, cancelErr)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)